src/python/ProcInfo.py

```diff
@@ -286,9 +286,9 @@ def diffWithOverflowCheck(this, new, old):
 		if new >= old:
 			return new - old;
 		else:
-			max = (1L << 31) * 2;  # 32 bits
+			max = (1 << 31) * 2;  # 32 bits
 			if old >= max:
-				max = (1L << 63) * 2;  # 64 bits
+				max = (1 << 63) * 2;  # 64 bits
```

Comment:
I will take care of Monalisa and movement to WMCore. For now I added to skip these files in CRABServer jenkins check